examples/Example2_PeriodicCommunications/Example2_PeriodicCommunications.ino

```diff
@@ -168,16 +168,6 @@ void loop()
     notecard.logDebug("performing sensor measurement\n");
     lastStatusMs = millis();
 
-    // Count the simulated measurements that we send to the cloud, and stop the
```

Comment:
There is no reason to count and stop, because this demo is based on button presses.